### ISSUE TITLE
New solution: One-line install, Homebrew integration, 6718 times smaller in code size

### ISSUE THREAD

**Issue by artginzburg** [mattrajca/sudo-touchid#31]
## TL;DR
As current solution became 5 years old, I propose the following:
[github.com/artginzburg/sudo-touchid](https://github.com/artginzburg/sudo-touchid#readme)

> Posting here to confirm stability over various systems. Works for me and my friends on macOS Catalina and Big Sur.

It's available through [Homebrew](https://brew.sh). Or without it.
Give it a star if it works for you.

## Long story
Solution proposed by @mattrajca is a fork, meaning it needs to be updated along with `sudo` itself. That's not quite scalable.
Also, the old solution is long. Literally long by meanings of both understanding and installation.

[The newly made solution](https://github.com/artginzburg/sudo-touchid#readme) is 6718 times smaller, which means it's so tiny that it could be read by anyone.
It's easily installed without building or using Xcode. This even allows to include it to your automated system build as a one-liner.
It concludes #10, #23, #29, and #30.

### What if I don't use Homebrew?
Then you're free to use an installation script utilising only built-in tools, which is also outlined in the repo.

> Homebrew provides stability, but is not necessary for the new solution to work.
